pkgs/applications/virtualization/qemu/default.nix

```diff
@@ -53,19 +54,19 @@ stdenv.mkDerivation (finalAttrs: {
     + lib.optionalString hostCpuOnly "-host-cpu-only"
     + lib.optionalString nixosTestRunner "-for-vm-tests"
     + lib.optionalString toolsOnly "-utils";
-  version = "8.1.3";
+  version = "8.2.0-rc3";
```

Comment:
`8.2.0-rc4` was released yesterday https://github.com/qemu/qemu/compare/v8.2.0-rc3...v8.2.0-rc4, if it's within the scope of this PR

Reply:
Indeed it is.  Updated.